### ISSUE TITLE
Open cloud audio file dialog: cannot be used by keyboard users

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#9862]
### Bug description

Using the keyboard, files cannot be selected, and so cannot be opened.

### Steps to reproduce

Using only the keyboard:
1. Open the "Open cloud audio file" dialog. The search box is the initial focus.
2. Tab to the list of audio files.

### Expected behavior

The first file is selected, and using the arrow keys you can select any file

### Actual behavior

No file is  selected, and it's not possible to select one to open.
Eg:

<img width="784" height="721" alt="Image" src="https://github.com/user-attachments/assets/93b85a77-99f7-4ece-bb78-38c3b68e5386" />

### Audacity Version

Other

### Operating system

Windows

### Additional context

This issue does not occur in the dialog for opening projects from the cloud.

**Comment by DavidBailes** [comment]
@dozzzzer . Please can you set a priority for this bug.

**Comment by dozzzzer** [comment]
@DavidBailes thanks for the bug, we'll fix it in 3.7.6

**Comment by dozzzzer** [comment]
@kryksyh 

When you go to the next page, the focus moves to the "Open" button every time you hit the Prev/Next buttons.

**Comment by dozzzzer** [comment]
Tested
